app/src/main/AndroidManifest.xml

```diff
@@ -80,6 +80,10 @@
             android:name=".synchronization.AppListActivity"
             android:configChanges="orientation|screenSize"></activity>
 
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="aa8a780391c17fadbc4d60a380447bbcac9ac150" />
```

Comment:
@jo-elimuai shouldn't we try to keep such keys secret?

Reply:
That's actually my personal Fabric API key.
I'll think about hiding it.

Reply:
The key should be hidden, yes. We could add the API key only for "release" builds. I'm working on a solution for this now.